dist/vidible-player-angular.js

```diff
@@ -161,8 +161,33 @@
 
             return VidibleLoaderQueue;
         }])
-        .directive('vidiblePlayer', ['$timeout', '$interval', 'vidibleQueueLoader',
-            function($timeout, $interval, vidibleQueueLoader) {
+        .factory('fullscreen',function() {
```

Comment:
Change name to VidibleArtifactsManipulation
We'll add here more Vidible html workarounds.